sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -24,6 +24,18 @@
 DEFAULT_KEY = 'Default'
 # This is useful if default value is desired when no match is found for lane speed key
 LANE_SPEED_DEFAULT_KEY = LANE_SPEED_KEY_PREFIX + DEFAULT_KEY
+
+# Standard media types:
+MEDIA_COPPER = 'COPPER'
+MEDIA_OPTICAL = 'OPTICAL'
+# Extended media types:
+EXTENDED_MEDIA_OPTICAL_LPO = MEDIA_OPTICAL + '_LPO'
```

Comment:
@longhuan-cisco Why can't it be just `MEDIA_LPO = 'LPO' `?

Reply:
Sure will update

Reply:
updated to `MEDIA_LPO = 'LPO'`